pkg/grpc/api.go

```diff
@@ -2030,8 +1952,6 @@ func (a *api) SubscribeConfigurationAlpha1(request *runtimev1pb.SubscribeConfigu
 	subscribeKeys := make([]string, 0)
 
 	// TODO(@halspang) provide a switch to use just resiliency or this.
```

Comment:
is this comment still valid?

Reply:
No clue what this even meant. @halspang ?

Reply:
ping @halspang